gobblin-hive-registration/src/main/java/org/apache/gobblin/hive/metastore/HiveMetaStoreRegisterWithoutComputingPartitionDiff.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.gobblin.hive.metastore;
+
+import com.google.common.base.Optional;
+import java.io.IOException;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.gobblin.configuration.State;
+import org.apache.gobblin.hive.HivePartition;
+import org.apache.hadoop.hive.metastore.IMetaStoreClient;
+import org.apache.hadoop.hive.metastore.api.AlreadyExistsException;
+import org.apache.hadoop.hive.metastore.api.Partition;
+import org.apache.hadoop.hive.metastore.api.Table;
+import org.apache.thrift.TException;
+
+
+@Slf4j
+public class HiveMetaStoreRegisterWithoutComputingPartitionDiff extends HiveMetaStoreBasedRegister {
+  public HiveMetaStoreRegisterWithoutComputingPartitionDiff(State state, Optional<String> metastoreURI) throws IOException {
+    super(state, metastoreURI);
+  }
+
+  @Override
+  protected void OnPartitionExist(IMetaStoreClient client, Table table, HivePartition partition,
+      Partition nativePartition, TException e) throws TException {
+    if(e instanceof AlreadyExistsException) {
+      log.info(String.format("Partition %s in table %s with location %s already exists and no need to update",
```

Comment:
this will end up flooding the log, suggest to use debug instead of info

Reply:
+1.